x-pack/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -25,145 +24,117 @@ import {
-            const newValue = field.value ?? '';
-            localField = {
-              field: field.field,
-              value: Array.isArray(newValue) ? newValue : [newValue],
-            };
-          }
-          return localField;
-        });
-
-        const colHeader = columnHeaders.find((col) => col.id === columnId);
-
-        const localLinkValues = getOr([], colHeader?.linkField ?? '', ecsData);
-
-        return (
-          <RenderCellValue
+    const gridColumns = useMemo(() => {
+      return getColumns(license);
+    }, [license]);
+
+    const columnHeaders = useMemo(() => {
+      return viewMode === VIEW_SELECTION.gridView ? gridColumns : eventRenderedViewColumns;
+    }, [gridColumns, viewMode]);
+
+    /**
+     * There is difference between how `triggers actions` fetched data v/s
+     * how security solution fetches data via timelineSearchStrategy
+     *
+     * _id and _index fields are array in timelineSearchStrategy  but not in
+     * ruleStrategy
+     *
+     *
+     */
+
+    const finalData = useMemo(() => {
+      return (data as TimelineNonEcsData[]).map((field) => {
+        if (['_id', '_index'].includes(field.field)) {
+          const newValue = field.value ?? '';
+          return {
+            field: field.field,
+            value: Array.isArray(newValue) ? newValue : [newValue],
+          };
+        } else {
+          return field;
+        }
+      });
+    }, [data]);
+
+    const actualSuppressionCount = useMemo(() => {
+      // We check both ecsData and data for the suppression count because it could be in either one,
+      // depending on where RenderCellValue is being used - when used in cases, data is populated,
+      // whereas in the regular security alerts table it's in ecsData
+      const ecsSuppressionCount = ecsData?.kibana?.alert.suppression?.docs_count?.[0];
+      const dataSuppressionCount = find({ field: 'kibana.alert.suppression.docs_count' }, data)
+        ?.value?.[0] as number | undefined;
+      return ecsSuppressionCount ? parseInt(ecsSuppressionCount, 10) : dataSuppressionCount;
+    }, [ecsData, data]);
+
+    const Renderer = useMemo(() => {
+      const myHeader = header ?? { id: columnId, ...browserFieldsByName[columnId] };
+      const colHeader = columnHeaders.find((col) => col.id === columnId);
+      const localLinkValues = getOr([], colHeader?.linkField ?? '', ecsData);
+      return (
+        <GuidedOnboardingTourStep
+          isTourAnchor={isTourAnchor}
+          step={AlertsCasesTourSteps.pointToAlertName}
+          tourId={SecurityStepId.alertsCases}
+        >
+          <DefaultCellRenderer
             browserFields={browserFields}
             columnId={columnId}
             data={finalData}
             ecsData={ecsData}
             eventId={eventId}
             header={myHeader}
             isDetails={isDetails}
-            isDraggable={isDraggable}
+            isDraggable={false}
```

Comment:
Is this correct? I see `isDraggable` is still a valid prop in `EuiDataGridCellProps['cellContext']` type